src/__tests__/operate.test.js

```diff
@@ -0,0 +1,8 @@
+import operate from '../logic/operate';
+
+test('should perform addition correctly', () => {
+  const addition = operate('5', '3', '+');
+  const subtraction = operate('5', '3', '-');
+  expect(addition).toEqual('8');
+  expect(subtraction).toEqual('2');
+});
```

Comment:
- [ ] Well done for covering some of the functionalities and scenarios of the `operate.js` module 👏. However, all the functionalities and scenarios need to be covered using Jest. Please consider writing more tests for "multiplication", "division", and "modulo division" 👍